css/index.css

```diff
@@ -158,18 +175,137 @@ header .logo {
 }
 
 .main-content .bottom-content p {
-    padding-right: 20px;
+    /*padding-right: 20px;*/
+    padding-right: 2rem;
 }
 
 /* Contact Styles */
 
 .contact p {
-    margin: 15px 0;
+    /*margin: 15px 0;*/
+    margin: 1.5rem 0;
 }
 
 /* Footer Styles */
 
 footer {
     text-align: center;
-    margin: 40px 0 20px;
+    /*margin: 40px 0 20px;*/
+    margin: 4rem 0 2rem;
+}
+
+/* RWD Solution */
+@media (max-width: 768px) {
+
+    .container {
+        width: 100%;
+    }
+
+    header nav {
+        /*width: 457px;*/
+        width: 59.5%;
+        /*margin-left: 20px;*/
+        margin-left: 3.6%;
+        margin-top: 5.5%;
+        font-size: 1.5rem;
+    }
+
+    header .logo {
+        /*margin: 28px 0 0 59px;*/
+        margin: 3.6% 0 0 7.7%;
+    }
+
+    .cta {
+        justify-content: center;
+    }
+    
+    .cta .cta-text {
+        /*margin: 0 40px 0 0;*/
+        margin: 0 5.2% 0 0;
+    }
+
+    .main-content .middle-img {
+       width: 100%;
+    }
+
+    .main-content {
+       /*margin: 75px 20px 35px;*/
+       margin: 9.7% 2.6% 4.5%;
+    }
+
+    .contact {
+        /*margin: 20px 20px 35px;*/
+        margin: 2.6% 2.6% 4.5%;
+    }
+}
+
+/* Phone */
+@media (max-width: 400px) {
+
+    header {
+        flex-wrap: wrap;
+        justify-content: center;
+    }
+
+    header nav {
+        width: 100%;
+    }
+
+    header nav {
+        width: 100%;
+        display: flex;
+        flex-direction: column;
+        margin: 0;
+        align-items: center;
+    }
+
+    header nav a {
+        width: 100%;
+        margin: 0;
+        display: block;
+        text-align: center;
+        padding: 5% 0;
+        font-size: 2rem;
+        border-bottom: 1px solid gray;
+    }
+
+    header nav a:first-child {
+        border-top: 1px solid gray;
+    }
+
+    header nav a:hover {
+        background: gray;
+    }
+
+    header .logo {
+        order: -1;
+        max-width: 100%;
+        margin: 5% 0;
+    }
+
+    .cta {
+        margin-top: 12.5%;
+        flex-flow: column;
+        align-items: center;
+    }
+    
+    .cta .cta-text {
+        margin: 0 0 12.5% 0;
+    }
+
+    .cta img {
+        max-width: 80%;
+    }
+
+    .main-content .top-content {
+        flex-wrap: wrap;
+        height: 75%;
+    }
+
+    .main-content .bottom-content {
+        flex-wrap: wrap;
+        align-content: space-between;
+        height: 110%;
+    }
+
```

Comment:
Overall, really solid job! Even getting the top/bottom margins to look good using percentages is a struggle and in you being able to do just that is great to see. Also, love seeing the fact that you commented your margins that were in px rather than deleting them whole. It is a good way to compare any changes that you would like to make! 👍